mautic/mautic
# Add default reply-to address in configuration

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? |  N
| New feature? | Y
| Automated tests included? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Add a field in Configuration > Email Settings for default reply to address 
 - Field is optional, if reply to email is empty, the sender address is used
 - If value is set on the individual email, that will override the global value
 - Send email action will need to account for this change

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  N/A

#### Steps to test this PR:
1. Navigate to Configuration > Email Settings
2. There should be a text field **Reply to address**
3. Field should not be required but should have email type validation.
4. On send mail action following should be expected:-
    - if reply to email is empty, the sender address should be used
    - if value is set on the individual email, that will override the global value

#### Other areas of Mautic that may be affected by the change:
1. 
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
